rules/windows/process_creation/proc_creation_win_exfil_data_via_cli.yml

```diff
@@ -0,0 +1,53 @@
+title: Possible Exfiltration Of Data Via CLI
+id: 7d1aaf3d-4304-425c-b7c3-162055e0b3ab
+status: experimental
+description: Detects the use of various cli utility related to web request exfiltrating data
+references:
+    - https://www.sentinelone.com/blog/living-off-windows-defender-lockbit-ransomware-sideloads-cobalt-strike-through-microsoft-security-tool/
+author: Nasreddine Bencherchali
+date: 2022/08/02
+tags:
+    - attack.execution
+    - attack.t1059.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_iwr:
+        Image|endswith:
+            - '\powershell.exe'
+            - '\pwsh.exe'
+            - '\cmd.exe'
+        CommandLine|contains:
+            - 'Invoke-WebRequest'
+            - 'iwr '
+            - 'wget '
+            - 'curl '
+        CommandLine|contains|all:
+            - ' -ur' # Shortest possible version of the -uri flag
+            - ' -me' # Shortest possible version of the -method flag
+            - ' -b'
+            - ' POST '
+    selection_curl:
+        Image|endswith: '\curl.exe'
+        CommandLine|contains|all:
+            - '--ur' # Shortest possible version of the --uri flag
+            - '-d' # Shortest possible version of the --data flag
+    payloads:
+        CommandLine|contains:
+            - 'ToBase64String'
+            - 'whoami'
+            - 'nltest'
+            - 'type'
```

Comment:
I'd exclude the `type` or combine it with a redirection ` >` / `>>` ... 🤔 I think I'd exclude it. 
It is too likely that it appears in a URL or filepath in a different context and causes false positives.

Reply:
You're right. I updated the rule